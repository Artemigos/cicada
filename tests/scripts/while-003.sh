echo hi

counter=18
while echo "$counter" | grep -iq "^1.$"
    counter=$(expr $counter + 1)

    if echo foo | grep -q foo
        echo found foo $counter
    fi
done

while echo "$counter" | grep -iq "^2[01]$"
    counter=$(expr $counter + 1)

    if echo foo | grep -q bar
        echo found bar
    else
        echo not found $counter
    fi
done

echo bye
