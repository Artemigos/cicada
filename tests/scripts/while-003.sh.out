hi
found foo 19
found foo 20
not found 21
not found 22
bye
